lemming_test.go

```diff
@@ -110,8 +111,9 @@ func TestStop(t *testing.T) {
 	}
 	f.gnmiServer.GetResponses = []interface{}{want}
 	cGNMI := gnmipb.NewGNMIClient(conn)
-	// Close the listener so the get must fail.
+	// Close the listener so the get must fail. Sleep to ensure listener is closed before Get.
 	f.GNMIListener().Close()
+	time.Sleep(100 * time.Millisecond)
```

Comment:
It looks like the reason is Close() is non-blocking. Since it's an interface anything can happen. I'm fine with deleting the cGNMI.Get() test.